waffle/models.py

```diff
@@ -229,7 +229,9 @@ def is_active_for_user(self, user):
         return None
 
     def _is_active_for_user(self, request):
-        return self.is_active_for_user(request.user)
```

Comment:
What about `return request is not None and self.is_active_for_user(request.user)`?

Reply:
One benefit of the getattr version is that it also works if request is not None, but user is not present at all in request. This can be the case if AuthenticationMiddleware is not loaded. In cases user is guaranteed to be set in request, `return request is not None and self.is_active_for_user(request.user)` would be sufficient of course.

Regarding tests: I am not quite sure how and where to add a test for this, maybe https://github.com/django-waffle/django-waffle/blob/master/waffle/tests/test_templates.py. How can we fake a None-request?

Reply:
Agreed on your implementation.

Ultimately this method is called by https://github.com/simonkern/django-waffle/blob/f198508571161eeef91b70b626b2f221f8631642/waffle/models.py#L244. Adding a test to https://github.com/simonkern/django-waffle/blob/f198508571161eeef91b70b626b2f221f8631642/waffle/tests/test_models.py should do the trick.